### PR TITLE
EL 6 - Adding support for password-auth

### PR DESCRIPTION
**Pull request by nbentoumi**
Pam config for RHEL6 uses both system-auth and password-auth but the module only manages system-auth

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -282,6 +282,41 @@ Content template of $system_auth_ac_file. If undef, parameter is set based on th
...
+----------------
+Path to password-auth. Used on RedHat.
+
+- *Default*: '/etc/pam.d/password-auth'
+
+password_auth_ac_file
+-------------------
+Path to password-auth-ac. Used on RedHat.
+
+- *Default*: '/etc/pam.d/password-auth-ac'
```
Comment (ghoneycutt) [inline README.md]: I don't see this file on EL6. Specifically CentOS 6.6

File: manifests/init.pp [hunk 2]
```diff
@@ -36,6 +36,12 @@
   $common_session_noninteractive_file  = '/etc/pam.d/common-session-noninteractive',
   $system_auth_file                    = '/etc/pam.d/system-auth',
   $system_auth_ac_file                 = '/etc/pam.d/system-auth-ac',
+  $password_auth_file                  = '/etc/pam.d/password-auth',
```
Comment (ghoneycutt) [inline manifests/init.pp]: all parameters need to be validated with corresponding spec tests

**Comment by ghoneycutt** [comment]
Hi @nbentoumi 

Thanks for your contribution and apologies for such a late response. Would you be interested in working with me with to get this finished and merged? I need to understand what `/etc/pam.d/password-auth-ac` is, get the params validated and their corresponding spec tests added.

**Comment by nbentoumi** [comment]
Hi @ghoneycutt 

Yes, we need to get this pull requested merged.
Pam config for RHEL6 uses both system-auth and password-auth but the module only manages system-auth.

"The problem with /etc/pam.d/system-auth is that it contains modules that are not usable in remote configurations so remote services such as sshd, vsftpd now use /etc/pam.d/password-auth." https://access.redhat.com/site/solutions/45492.

Thanks for help

**Comment by ghoneycutt** [comment]
@nbentoumi will be happy to merge once it is ready with tests. Could you please answer the questions that are inline with the code?

**Comment by nbentoumi** [comment]
Will do another pull request

**Pull request closed without merging**